api/v1alpha1/stage_types.go

```diff
@@ -670,6 +672,15 @@ type PromotionInfo struct {
 	Freight FreightReference `json:"freight" protobuf:"bytes,2,opt,name=freight"`
 }
 
+type LastPromotionInfo struct {
+	// Name is the name of the Promotion
+	Name string `json:"name" protobuf:"bytes,1,opt,name=name"`
+	// Freight is the freight being promoted
+	Freight FreightReference `json:"freight" protobuf:"bytes,2,opt,name=freight"`
+	// Status is the status of the promotion
+	Status PromotionStatus `json:"status" protobuf:"bytes,3,opt,name=status"`
```

Comment:
I wonder if we could just add this field to the existing `PromotionInfo` type instead of making a whole new type that so closely resembles it.